src/components/candidates/Candidate.vue

```diff
@@ -90,8 +112,14 @@ function deleteProfile() {
           />
           {{ currentCandidate.linkedinUrl }}
         </div>
-        <SubmitButton @click="editProfile" />
-        <DeleteButton @click="deleteProfile" />
+        <SubmitButton
```

Comment:
The text should be `Edit`
![Screenshot 2022-11-30 at 12 20 45](https://user-images.githubusercontent.com/10779650/204757282-ccbe99ef-1db1-46fe-be5a-2e4ffc07bd65.jpg)